osquery/tables/system/windows/nt_info.cpp

```diff
@@ -0,0 +1,47 @@
+/**
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under both the Apache 2.0 license (found in the
+ *  LICENSE file in the root directory of this source tree) and the GPLv2 (found
+ *  in the COPYING file in the root directory of this source tree).
+ *  You may select, at your option, one of the above-listed licenses.
+ */
+
+#include <osquery/logger.h>
+#include <osquery/sql.h>
+#include <osquery/system.h>
+#include <osquery/tables.h>
+
+#include "osquery/core/conversions.h"
+#include "osquery/core/windows/wmi.h"
+
+namespace osquery {
+namespace tables {
+
+QueryData genNtInfo(QueryContext& context) {
+  Row r;
+  QueryData results;
+
+  WmiRequest wmiSystemReq("select * from Win32_NtDomain");
+  std::vector<WmiResultItem>& wmiResults = wmiSystemReq.results();
```

Comment:
I can not see the reason to modify the `wmiResults` variable after getting it from `results`. So, it would be great to make it `const`. Smth like:
```c++
const auto& wmiResults = wmiSystemReq.results();
```